webapp/src/components/OrderDetail.jsx

```diff
@@ -105,6 +105,10 @@ function FulfillmentOption({ order, onOrderUpdated }) {
   const [optionId, setOptionId] = React.useState(0);
   const { showErrorToast } = useErrorToast();
 
+  if (!order.fulfillmentOption) {
```

Comment:
Someone still needs to be able to edit if there is no chosen fulfillment option, but there are options to choose. Maybe you want this inside the `isEmpty`check on line 112?

Reply:
This made sense, and works like we expect now. Thanks.